### PR TITLE
correct the link of good first issues contribution website

### PR DESCRIPTION
**Pull request by sheoranharis**

### CODE REVIEW

File: Open-Source/Contribute/Readme.md [hunk 1]
```diff
@@ -22,7 +22,7 @@
 - [EddieBot](https://github.com/EddieHubCommunity/EddieBot)
 - [facebook-chat-api](https://github.com/Schmavery/facebook-chat-api)
 - [Falso.js](https://github.com/ngneat/falso)
-- [first-contributions](https://github.com/firstcontributions/first-contribution)
+- [first-contributions](https://firstcontributions.github.io)
```
Comment (kaiwalyakoparkar) [inline Open-Source/Contribute/Readme.md:25]: Please do not change the entire link. These two seems to be two separate repos

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

**Comment by kaiwalyakoparkar** [comment]
Hey @sheoranharis it's a good practice to show your interest to work on an issue. I am closing this pr as this issue has been assigned to someone else 😞 . Hope you understand. There are some unassigned issues you can look forward to contribute to 🚀

**Comment by sheoranharis** [comment]
Ok

**Comment by sheoranharis** [comment]
Where can I add learning resources for full stack web development?

**Comment by kaiwalyakoparkar** [comment]
@sheoranharis you can add them into the discussion for the particular tech stack and we will get it added

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Pull request closed without merging**